Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -305,6 +311,7 @@ - (void)videoPlaybackPaused:(NSDictionary *)properties withOptions:(NSDictionary
 
     // [self.streamAnalytics.configuration addLabels:map]; TBD if needed
     [self.streamAnalytics setMetadata:playbackMetaData];
+    // [self.configurationLabels setDictionary:map]; TBD if needed
```

Comment:
@brennan is the plan to have Fox test this branch to validate the need for this or are we awaiting conformation from Comscore contacts?

Reply:
Line 312 and 314 are complementary - line 312 was already commented out in the prior version of this SDK, so added line 314. If needed, we can uncomment line 312 and line 314. That said, not sure if/what we're waiting on before we can do so.

Reply:
I've asked Fox to review/test this branch as is and let us know if they find any issues. That said, Comscore and Fox did a thorough review of our mobile integrations earlier this year and shared all feedback at that point. They did not surface anything related to labels and the Video Playback Paused event so I would assume if line 312 was already commented out, we should leave as is. What exactly would those lines do? 

Also, it doesn't look like we set labels in the [Android integration](https://github.com/segment-integrations/analytics-android-integration-comscore/blob/master/src/main/java/com/segment/analytics/android/integrations/comscore/ComScoreIntegration.java) for Video Playback Paused